homeassistant/components/plex/media_browser.py

```diff
@@ -94,31 +94,30 @@ def build_item_response(payload):
     if special_folder:
         if media_content_type == "server":
             library_or_section = plex_server.library
-            children_media_class = MEDIA_CLASS_DIRECTORY
+            media_class = MEDIA_CLASS_DIRECTORY
             title = plex_server.friendly_name
         elif media_content_type == "library":
             library_or_section = plex_server.library.sectionByID(media_content_id)
             title = library_or_section.title
             try:
-                children_media_class = ITEM_TYPE_MEDIA_CLASS[library_or_section.TYPE]
+                media_class = ITEM_TYPE_MEDIA_CLASS[library_or_section.TYPE]
```

Comment:
A library of movies should still be directory and have children_media_class set to movie.

Reply:
See eg the spotify integration:
https://github.com/home-assistant/core/blob/8a16fb53c8beb41c35196f1343b07ad8927b9d97/homeassistant/components/spotify/media_player.py#L107-L159